extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -15,4 +15,11 @@
  */
 
  amp-sticky-ad {
+ 	position: fixed;
+ 	bottom: 0;
+ 	left: 0 !important;
+ 	z-index: 1;
+ 	width: 100%;
+ 	display: none;
```

Comment:
What's the actual `display` of an `amp-sticky-ad` element? Generally, `display:none` not needed here if they are required to be `layout=nodisplay`.